tests/test_sensor.py

```
@@ -2476,82 +2703,48 @@ def test_smart_sensor_finds_device_after_ignored_rows(
-        ({}, "smart.nvme0.temperature"),
-        (False, "smart.nvme0.temperature"),
-        ("unknown", "smart.nvme0.temperature"),
+        [],
+        {},
+        False,
+        {"current": False},
+        {"current": "unknown"},
     ],
 )
 def test_smart_sensor_unavailable_when_property_malformed(
     value: Any,
```

Ruff (0.15.17)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)